docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -62,43 +62,122 @@ and details on the intrapartum and neonatal time periods.
 
-**Wave 1**: Add new risk and cause models and update existing risks and causes as needed
+* Wave 1 will include in basic model design, outlines of the healthcare system, and some interventions (AI ultrasound, higher level delivery facility interventions, RDS management). 
+* Wave 2 will some antenatal supplements(MMS, IV iron), the hemoglobin risk for birthing parents, and all downstream causes affected by hemoglobin. 
+* Wave 3 will add in gestational blood pressure and relevant causes and risks including pre-eclampsia care and downstream effects of high blood pressure. 
 
-.. todo::
+**Wave 1 Concept Model Map:**
 
-  Determine if wave 1 will include making multiple, smaller simulations covering a subset of risks and causes or if it will all be integrated.
+.. image:: wave_1_full.svg
 
-- Wave 1 will be focused on expanding the risks and causes to cover everything we need to model the new interventions and calibrating a baseline model 
-- There will be only the baseline scenario in this wave 
-- We will make the simulation 3 time steps, but without the healthcare system or decision trees at each time step 
-- Making the model 3 time steps will necessitate us to tackle issues around YLDs and disability weights disconnected from "time" as we normally think of it in the simulation 
-
-**Wave 2**: Add in healthcare during pregnancy and related interventions
-
-- Wave 2 will add the first decision tree for the healthcare system during pregnancy 
-- We will add interventions that are administered during pregnancy and primarily impact the health of the birthing parent and previously modeled child impacts (MMS, BEP, IV iron, anemia diagnostics and treatment, pre-ecamplsia diagnostics and treatment, and antenatal azithromycin)
-- Any downstream effects on antenatal or child outcomes can be modeled with dummy impact variables if the impacts cannot be properly included yet (e.g., since deliver in a higher level facility is not included yet, we can include a dummy variable for birth impacts)
-- Additionally, we will decide on how to include the post-neonatal child effects. We anticipate adding a summarized later childhood impact (post neonatal to age 5) based on the nutrition optimization work, but this has not been decided.
-
-**Wave 3**: Add the intrapartum model and related interventions
-
-- Wave 3 will add the second decision tree for the intrapartum time period 
-- We will add interventions that are administered during OR affect the intrapartum period (AI ultrasound, birth monitoring and sensors, PPH bundle)
-- Any downstream effects on neonatal or later child outcomes can be modeled with with dummy impact variables if the impacts cannot be properly included yet
+.. _mncnh_portfolio_3.1:
 
-**Wave 4**: Add the neonatal period and related interventions
+3.1 Model Components
+--------------------
+
+Our model will have 3 main "components" that each represent a part of a 
+parent and child dyad's journey. In this model, the simulants (note: simulant 
```

Comment:
```suggestion
journey of a parent-child dyad. In this model, the simulants (note: simulant 
```

Reply:
If you go with Abie's suggestion, I'd also change the last "a" in the previous line to "the," so that it reads "Our model will have 3 main "components" that each represent a part of **the** journey of a parent-child dyad."